### PR TITLE
increase chunk size

### PR DESCRIPTION
**Pull request by sahiga**

### CODE REVIEW

File: js/src/chunking/DefaultChunkingStrategy.coffee [hunk 1]
```diff
@@ -5,7 +5,7 @@ define 'kryptnostic.chunking.strategy.default', [
 
   _ = require 'lodash'
 
-  BLOCK_LENGTH_IN_BYTES = 4096
+  BLOCK_LENGTH_IN_BYTES = 1000000
```
Comment (makosblade) [inline js/src/chunking/DefaultChunkingStrategy.coffee]: Can we maybe make this a power of two? like 1,048,576

File: js/test/chunking/DefaultChunkingStrategy-test.coffee [hunk 2]
```diff
@@ -1,11 +1,11 @@
 # mock data
 # =========
 
-DATA_LONG = [0...8000].map((i) -> 'a').join('')
+DATA_LONG = [0...1000200].map((i) -> 'a').join('')
```
Comment (makosblade) [inline js/test/chunking/DefaultChunkingStrategy-test.coffee]: We should pull these out into constants in this test at some point

File: js/src/chunking/DefaultChunkingStrategy.coffee (hunk 1 above)
Comment (nickdhewitt) [inline js/src/chunking/DefaultChunkingStrategy.coffee]: It's not required to be a power of two on the backend. Any reason you prefer that?

Comment (makosblade) [inline js/src/chunking/DefaultChunkingStrategy.coffee]: Just to satisfy my inner memory sizing nerd.

**Pull request merged**